### ISSUE TITLE
blockOuterScroll behaviour - scroll is jumping

### ISSUE THREAD

**Issue by vanya6537** [rommguy/react-custom-scroll#107]
Hello!

Scroll is jumping sometimes in our portal. 
I tried to investigate what causes this behaviour.

https://github.com/rommguy/react-custom-scroll/blob/1f4757198e2478090ac21bcd2849ae388f3afc60/src/main/customScroll.js#L277

Changing this line to `const delta = e.deltaY` can fix this behaviour.

Please explain what is behind this mathematics(% 3 and * 10).

**Comment by cuteapoot** [comment]
This issue would explain the problem I'm currently having with this package. Any reason this issue has been untouched for over a year? It seems like this is a problem that would be affecting everyone using this package, so it's strange to me that it hasn't been addressed.

**Comment by rommguy** [comment]
@cuteapoot The problem of blocking outer scroll was very difficult to solve, as I needed to identify this scroll event is about to reach the end of the scroll area, **before applying the scroll** and block it.      
At the time, calculating how much pixels will be scrolled required the % 3 check to identify Explorer browser, which used different units. 

I just found that there's a css property that didn't exist when I created the library called "overscroll-behavior-block" and it seems it is supported in all relevant browsers.

I'll try to implement it instead of this calculation and publish a fix
